src/core/server/capabilities/routes/resolve_capabilities.ts

```diff
@@ -29,14 +29,18 @@ export function registerCapabilitiesRoutes(router: IRouter, resolver: Capabiliti
         authRequired: 'optional',
       },
       validate: {
+        query: schema.object({
+          useDefaultCapabilities: schema.boolean({ defaultValue: false }),
+        }),
         body: schema.object({
           applications: schema.arrayOf(schema.string()),
         }),
```

Comment:
NIT: any reason to introduce the new `useDefaultCapabilities` in `query` instead of `body`?

Reply:
No real reason. It "felt" more appropriate to make it a query parameter, but I'll defer to you here. If you'd prefer a `body` parameter instead, then I'll happily make that change.

Reply:
na, it's probably fine